src/sdk/sts_sdk.js

```diff
@@ -0,0 +1,100 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const cloud_utils = require('../util/cloud_utils');
+const dbg = require('../util/debug_module')(__filename);
+const LRUCache = require('../util/lru_cache');
+const { RpcError } = require('../rpc');
+const signature_utils = require('../util/signature_utils');
+const db_client = require('../util/db_client').instance();
+
+// DO we need that cache here? 
+const account_cache = new LRUCache({
```

Comment:
can we combine this with the object_sdk cache?

Reply:
another thing I discussed with @jeniawhite and we had some doubts about, do we want to share it with object_sdk? 
do we maybe want a different cache with a different expiry here since most of our operations in this sdk are on accounts

Reply:
I would prefer a single cache if possible. What would be the different expiry that you had in mind?

Reply:
sure, I eventually used the object_sdk accounts cache instead of this new one, I think that for now it's enough to have 1 and if in the future we will decide to do optimizations of the account cache for sts purposes we can split them. 
but regarding the expiry, I thought since we are mainly using the accounts info here we need shorter expirty.